data/35e/wizards_of_the_coast/supplement/book_of_vile_darkness/bv_spells.lst

```diff
@@ -2,13 +2,23 @@
 Aberrate															TYPE:Arcane			CLASSES:Sorcerer,Wizard=1						SCHOOL:Transmutation					DESCRIPTOR:Evil				COMPS:V, S, Fiend			CASTTIME:1 standard action	RANGE:Touch		TARGETAREA:one living creature																DURATION:10 minutes/level				SAVEINFO:Fortitude negates			SPELLRES:Yes												SOURCEPAGE:BoVD	DESC:Subject becomes an aberration.
 Abyssal Might														TYPE:Arcane.Divine	CLASSES:Blackguard=3|Cleric,Sorcerer,Wizard=4			SCHOOL:Conjuration	SUBSCHOOL:Summoning	DESCRIPTOR:Evil				COMPS:V, S, M, Demon		CASTTIME:1 standard action	RANGE:Personal	TARGETAREA:You																			DURATION:10 minutes/level				SAVEINFO:None					SPELLRES:No													SOURCEPAGE:BoVD	DESC:Caster gains +2 to STR, CON, DEX and SR,.
 Addiction															TYPE:Arcane.Divine	CLASSES:Assassin=1|Bard,Cleric,Sorcerer,Wizard=2		SCHOOL:Enchantment											COMPS:V, S, Drug			CASTTIME:1 standard action	RANGE:Touch		TARGETAREA:one living creature																DURATION:Instantaneous					SAVEINFO:Fortitude negates			SPELLRES:Yes												SOURCEPAGE:BoVD	DESC:Subject becomes addicted to a drug. Drug:the chosen drug for the addiction.
 Alert Bebilith														TYPE:Arcane			CLASSES:Sorcerer,Wizard=6						SCHOOL:Conjuration	SUBSCHOOL:Calling		DESCRIPTOR:Evil				COMPS:V, S, M			CASTTIME:1 standard action	RANGE:Close		TARGETAREA:One bebilith																		DURATION:Instantaneous					SAVEINFO:None					SPELLRES:No													SOURCEPAGE:BoVD	DESC:Summons bebilith to deal with a demon.
 Angry Ache															TYPE:Divine			CLASSES:Assassin,Cleric=1						SCHOOL:Necromancy												COMPS:V, S				CASTTIME:1 standard action	RANGE:Close		TARGETAREA:one living creature																DURATION:1 minute/level					SAVEINFO:Fortitude negates			SPELLRES:Yes												SOURCEPAGE:BoVD	DESC:Subject takes -2 penalty on attack rolls.
-Befoul															TYPE:Divine			CLASSES:Cleric=8								SCHOOL:Transmutation					DESCRIPTOR:Evil				COMPS:V, S, M			CASTTIME:1 minute			RANGE:Touch		TARGETAREA:Water in a volume of 100-ft/level by 100-ft/level by 10-ft/level									DURATION:Permanent (see text)				SAVEINFO:None (see text)			SPELLRES:No													SOURCEPAGE:BoVD	DESC:Large amount of water becomes poisonous.
+# listed below
+# Did I miss something?
+#Befoul															TYPE:Divine			CLASSES:Cleric=8								SCHOOL:Transmutation					DESCRIPTOR:Evil				COMPS:V, S, M			CASTTIME:1 minute			RANGE:Touch		TARGETAREA:Water in a volume of 100-ft/level by 100-ft/level by 10-ft/level									DURATION:Permanent (see text)				SAVEINFO:None (see text)			SPELLRES:No													SOURCEPAGE:BoVD	DESC:Large amount of water becomes poisonous.
```

Comment:
Why are we commenting out a legit spell?

![image](https://user-images.githubusercontent.com/4307487/46121604-601e8400-c1c9-11e8-893c-99f367918752.png)

Reply:
Sorry, that comment should have been removed. Befoul appears twice in the LST; once where I commented it and afterward in the corrupt list.
Instead of commenting it out, I should have removed it.